test/test_browser.py

```diff
@@ -3422,6 +3422,17 @@ def test_async_returnvalue(self, args):
       create_file('filey.txt', 'sync_tunnel\nsync_tunnel_bool\n')
     self.btest('browser/async_returnvalue.cpp', '0', args=['-sASYNCIFY', '-sASYNCIFY_IGNORE_INDIRECT', '--js-library', test_file('browser/async_returnvalue.js')] + args + ['-sASSERTIONS'])
 
+  @parameterized({
+    'normal': (['-sASYNCIFY_EXPORTS=*nsuspendExp*'],), # noqa
+    'nothing': (['-DBAD'],), # noqa
+  })
+  def test_async_exports(self, args):
+    if not is_chrome():
+      # self.skipTest(f'Current browser ({EMTEST_BROWSER}) does not support JSPI. Only chromium-based browsers ({CHROMIUM_BASED_BROWSERS}) support JSPI today.')
+      # TODO(mboc-qt): Use the above once browser_test_add_opera/1 is merged.
+      self.skipTest(f'Current browser ({EMTEST_BROWSER}) does not support JSPI.')
+    self.btest('browser/async_exports.cpp', '0', args=['-sASYNCIFY=2', '-Wno-experimental'] + args + ['-sASSERTIONS'])
```

Comment:
Does this need to be a browser test?   Can it go in `test_other.py` or `test_core.py`?   @brendandahl WDYT?  Does this require a completely new test or can we modify an existing one?

Reply:
`test_other.py` is probably more appropriate (see `test_embind_jspi` for an example).

Reply:
Tried moving it there, but see my other comment below - no easy way to check if a function has been instrumented or not.

Reply:
Moved to test_other with help from @brendandahl